userspace/engine/rule_loader_reader.cpp

```diff
@@ -319,8 +319,10 @@ static void read_rule_exceptions(
 		// Check if an exception with the same name has already been defined
 		for (auto &exception : exceptions)
 		{
-			THROW((v_ex.name == exception.name),
-			       "Exceptions names in the same object must be unique", ex_ctx);
+			if(v_ex.name == exception.name)
+			{
+				cfg.res->add_warning(falco::load_result::LOAD_EXCEPTION_NAME_NOT_UNIQUE, "Multiple exceptions defined with the same name", ex_ctx);
```

Comment:
```suggestion
				cfg.res->add_warning(falco::load_result::LOAD_EXCEPTION_NAME_NOT_UNIQUE, "Multiple definitions of exception '" + v_ex.name + "' in the same rule", ex_ctx);
```
Suggesting a more semantic error.